### ISSUE TITLE
Final directory for css

### ISSUE THREAD

**Issue by simonh1000** [archfirst/angular2-seed-sass#1]
Have you been able to solve the problem that seems to have been left by https://github.com/mgechev/angular2-seed/wiki/Add-custom-Gulp-task in that the final css is in the `src` directory and not in `dist`?

**Comment by nareshbhatia** [comment]
Nope. I tried 3 options:
1. Use Sass files in `styleUrls` and in-line them during the build. This made the build very complex and deviate heavily from angular2-seed. This would be difficult to keep in sync with the parent project.
2. Use a temp directory for all preprocessed files and run the dev environment from this directory. Unfortunately, this would mean copying everything to the temp directory, increasing the build time and making the dev process very inefficient.
3. Preprocess Sass files in place, leaving them in the src directory. I add `.css` to `.gitignore` so that these files don't get checked in accidentally. It also assumes that we don't have `.css` in source - which seems to be a reasonable assumption if we are using Sass anyway.

I finally settled on solution 3. It is a compromise, but better that solutions 1 and 2, I feel.
